src/operators/expand-support.ts

```diff
@@ -41,24 +58,31 @@ export class ExpandSubscriber<T, R> extends OuterSubscriber<T, R> {
     }
 
     const index = this.index++;
-    if (this.active < this.concurrent) {
+    if (this.active < this._concurrent) {
       destination.next(value);
       let result = tryCatch(this.project)(value, index);
       if (result === errorObject) {
         destination.error(result.e);
+      } else if (!this.scheduler) {
+        this._innerSub(result, value, index);
       } else {
-        if (result._isScalar) {
-          this._next(result.value);
-        } else {
-          this.active++;
-          this.add(subscribeToResult<T, R>(this, result, value, index));
-        }
+        const state = { subscriber: this, result, value, index };
+        this.add(this.scheduler.schedule(disptch, 0, state));
       }
     } else {
       this.buffer.push(value);
     }
   }
 
+  _innerSub(result: any, value: any, index: number): void {
```

Comment:
I don't think so in general in RxJS codebases so far I know. Patterns I could see was some of inherited (such as `_next`, etcs) are starts with underbar, along with some private fields. Just thought in typical cases of instance method wouldn't need to start with underbar, especially `_innerSub` is public and being called externally via `dispatch`.